nextflow_schema.json

```diff
@@ -92,37 +116,41 @@
                 },
                 "min_sv_size": {
                     "type": "integer",
-                    "description": "Minimum SV size of variants to benchmark, 0 to disable , Default:30",
+                    "description": "Minimum SV size of variants to benchmark, 0 to disable",
+                    "default": 0,
```

Comment:
previously 30 was the defult, now its 30? Just in case its an oversight ;)

Reply:
That is a good catch, It is a misunderstanding actually. Tools uses 30 as default, and I am disabling it by default using 0.